### DISCUSSION TITLE
unable to locate resource: esapi-java-logging.properties

### DISCUSSION THREAD

**Discussion by RutujaPSonawane** [ESAPI/esapi-java-legacy#765] [Q&A]
I upgrade the esapi version from 2.2.0.0 to 2.5.1.0 and I have added the esapi-java-logging.properties file but still its throwing error like unable to locate resource: esapi-java-logging.properties

**Comment by jeremiahjstacey** [comment]
Please use discussions or mailing lists to ask questions about your specific implementation issues.  This is not a bug with ESAPI, but rather an issue you are encountering with your environment.

Please read all release notes related to the versions you are upgrading to and from, and also ensure that all properties that are present in the latest 2.5.1.0 configuration file are both present and configured in your environment.

**Reply by jeremiahjstacey** [reply]
Release notes can be reviewed [HERE](https://github.com/ESAPI/esapi-java-legacy/releases)

The latest ESAPI.properties can be inspected from the current baseline [HERE](https://github.com/ESAPI/esapi-java-legacy/blob/develop/configuration/esapi/ESAPI.properties)
OR 
Reviewed from the bundle attached to the 2.5.1.0 release scope [HERE](https://github.com/ESAPI/esapi-java-legacy/releases/download/esapi-2.5.1.0/esapi-2.5.1.0-configuration.jar)
